class AlreadyExistsExc(Exception):
    pass


class ObjectNotFoundExc(Exception):
    pass
